pkg/steps/multi_stage.go

```diff
@@ -565,7 +577,13 @@ func (s *multiStageTestStep) generatePods(steps []api.LiteralTestStep, env []cor
 			addProfile(s.profileSecretName(), s.profile, pod)
 		}
 		if step.Cli != "" {
-			addCliInjector(step.Cli, pod)
+			release := step.Cli
+			if s.clusterClaim != nil {
+				if claimRelease.OverrideName == release {
```

Comment:
Could we use DependencyParts here? Maybe call with `fmt.Sprintf("%s:cli", step.Cli)`?